src/chttpd_auth_cache.erl

```diff
@@ -209,3 +206,18 @@ update_doc_ignoring_conflict(DbName, Doc, Options) ->
         throw:conflict ->
             ok
     end.
+
+validate_user_creds(nil) ->
+    nil;
+validate_user_creds(UserCreds) ->
+    couch_log:info("~n~n validate_user_creds: ~p~n~n", [UserCreds]),
```

Comment:
Agree. Logging on info will cause information leak though logs and systems they analyse in something more than developer environment.